components/brave_ads/browser/buildflags/buildflags.gni

```diff
@@ -2,5 +2,7 @@ import("//brave/components/brave_rewards/browser/buildflags/buildflags.gni")
 import("//build/config/features.gni")
 
 declare_args() {
-  brave_ads_enabled = brave_rewards_enabled && (is_mac || is_linux || is_win || is_android)
+  brave_ads_enabled =
+      brave_rewards_enabled && (is_mac || is_linux || is_win || is_android)
+  enable_native_notifications = false
```

Comment:
this should also go away because it will not work correctly without the corresponding changes in upstream